src/Symfony/Bridge/Monolog/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ CHANGELOG
 -----
 
  * added `ProcessorInterface`: an optional interface to allow autoconfiguration of Monolog processors
+ * [DEPRECATION] add trigger_error in case of inherit class using getLogs and countErrors 
```

Comment:
no need for the `[...]` label
also the message should be less technical and more useful to the reader (stating roughly the same as the deprecation message)